bundle/language_en.properties

```diff
@@ -1,4 +1,74 @@
+# Common dialog titles
+angal.common.error.title                                                         = Error
+angal.common.warning.title                                                       = Warning
+angal.common.info.title                                                          = Information
+angal.common.question.title                                                      = Question
+angal.common.plain.title                                                         = Note
 
+# Common button text and short cut keys
+angal.common.cancel.btn                                                          = Cancel
+angal.common.cancel.btn.key                                                      = C
+angal.common.close.btn                                                           = Close
+angal.common.close.btn.key                                                       = C
+angal.common.delete.btn                                                          = Delete
+angal.common.delete.btn.key                                                      = D
+angal.common.edit.btn                                                            = Edit
+angal.common.edit.btn.key                                                        = E
+angal.common.new.btn                                                             = New
+angal.common.new.btn.key                                                         = N
+angal.common.ok.btn                                                              = OK
+angal.common.ok.btn.key                                                          = O
+angal.common.submit.btn                                                          = Submit
+angal.common.submit.btn.key                                                      = S
+
+# Other common messages and text
+angal.common.select.row.msg                                                      = Please select a row.
+angal.common.data.not.saved.msg                                                  = The data could not be saved.
+
+angal.common.userid.label                                                        = Userid:
```

Comment:
I will change to "User".

As for colons for labels I like them and most "standards" use them too.   Look at a Find or Replace dialog for example.

But I will do whatever the project dictates.

Reply:
> As for colons for labels I like them and most "standards" use them too. Look at a Find or Replace dialog for example.

Yes, most modern styles don't even put a label, like Thunderbird:
![image](https://user-images.githubusercontent.com/2938553/117358394-64999600-aeb6-11eb-80f7-1106de056e3b.png)

or Chrome:
![image](https://user-images.githubusercontent.com/2938553/117359126-441e0b80-aeb7-11eb-9741-3594ea437215.png)

But I'm thinking about translators and the way the [Transifex Translation Memory](https://docs.transifex.com/translation-memory/tm-introduction) (TM) will treat them because I'm not sure yet if it cares about punctuation or not.

So the point is: in cases they are present, are the colons "part of" the label OR are they added programmatically to the localized message?

Reply:
I can find counter examples all over; this one is from Outlook on Windows:
![image](https://user-images.githubusercontent.com/1105445/117361620-43ba4c00-ae88-11eb-8097-4f094e94480a.png)

Linux Text Edit
![image](https://user-images.githubusercontent.com/1105445/117361890-95fb6d00-ae88-11eb-87bb-ee7d8a71e0d4.png)

Linux Default Printer dialog
![image](https://user-images.githubusercontent.com/1105445/117362022-c2af8480-ae88-11eb-8f22-a71daf374915.png)

My default behavior and previous projects is to require the colons as part of the label but if the only way I'll get them is to add them programmatically then so be it 😄